cobbler.spec

```diff
@@ -298,7 +307,7 @@ echo "ERROR: DOCPATH: ${DOCPATH} does not match %{_mandir}"
 
 %install
 . distro_build_configs.sh
-# bypass install errors ( don't chown in install step)
+# bypass install errors (don't chown in install step)
 %py3_install ||:
```

Comment:
The comment:
# bypass install errors (don't chown in install step)
can vanish, together with:
||:
in the next line.
This was because setup.py install returned with an error code, due to bad chown calls.
The ||: syntax simply ignores all errors from setup.py install
You should be able to integrate this. I can do this separately if you are unsure.

Reply:
Done.